.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
This file is actually supported by majority of editors and IDEs (check editorconfig.org), so I thought it would be handy to keep such a configuration, to ensure uniform formatting with future contributions. But can remove it if it's not needed, of course.